src/ol/imagebase.js

```diff
@@ -24,7 +24,7 @@ ol.ImageState = {
  * @constructor
  * @extends {goog.events.EventTarget}
  * @param {ol.Extent} extent Extent.
- * @param {number|undefined} resolution Resolution.
+ * @param {Array.<number>|undefined} resolution Resolution.
```

Comment:
Maybe add something like "First value is the x-resolution, second the y-resolution".